incubator/webpagetest-agent/README.md

```diff
@@ -0,0 +1,107 @@
+
+
+> **Tip**: List all releases using `helm list`
+
+## Uninstalling the Chart
+
+To uninstall/delete the `my-release` deployment:
+
+```console
+$ helm delete my-release
+```
+
+The command removes all the Kubernetes components associated with the chart and deletes the release.
+
+## Configuration
+
+The following tables lists the configurable parameters of the WordPress chart and their default values.
+
+| Parameter                            | Description                                | Default                                                    |
+| -------------------------------      | -------------------------------            | ---------------------------------------------------------- |
+| `image`                              | WebPageTest server image                   | `webpagetest/server:{VERSION}`                              |
+| `imagePullPolicy`                    | Image pull policy                          | `IfNotPresent`                                             |
+| `agent.key`                          | API Key for agent to auth the the master   | `nil`                                                      |
+| `agent.server`                       | Hostname of the WPT server (see below)     | `agent-web-page-test.local`                                |
+| `agent.location`                     | WPT location tag                           | `Test`                                                     |
+| `agent.shaper`                       | Traffic shaping method the agent uses      | `none`                                                      |
+| `nodeSelector`                       | Node labels for pod assignment             | `{}`                                                       |
+
+
+Specify each parameter using the `--set key=value[,key=value]` argument to `helm install`. For example,
+
+```console
+$ helm install --name my-release \
+  --set agent.key=SecretKey \
+    incubator/webpagetest-agent
+```
+
+Alternatively, a YAML file that specifies the values for the above parameters can be provided while installing the chart. For example,
+
+```console
+$ helm install --name my-release -f values.yaml incubator/webpagetest-agent
+```
+
+> **Tip**: You can use the default [values.yaml](values.yaml)
+
+
+## Ingress
+There are none. This is a worker node. the standard operation is for it to poll the
+WebPageTest Server for work.
+
+## WebPageTest Server Config
+At time of writing this chart the server has two means of config
+* Log onto the server and use vim or similar
+* Use the AWS/EC2 init scripts to configure some values, the login and use vim ir similar to configure the rest
+As such these charts will get an environment up and working, but if you want more than defaults you will need to write your own config maps or update the WebPageTest Source for either the [server](https://github.com/WPO-Foundation/webpagetest) or the [agent](https://github.com/WPO-Foundation/wptagent)
```

Comment:
Could you provide an example configmap or a default one for the case this isn't in AWS/EC2. We shouldn't assume AWS.

Reply:
I'm sorry I don't know the config structure well enough for that.